app/models/trip.rb

```diff
@@ -0,0 +1,22 @@
+include ActionView::Helpers::NumberHelper
+
+class Trip < ApplicationRecord
+  belongs_to :driver
+  belongs_to :passenger
+
+  validates :driver_id, :passenger_id, :date, :cost, presence: true
+  validates :rating, inclusion: { in: [nil, 1, 2, 3, 4, 5] }
+  validates :cost, numericality: true
+
+  def driver
+    return Driver.find_by(id: self.driver_id)
+  end
+
+  def passenger
+    return Passenger.find_by(id: self.passenger_id)
+  end
```

Comment:
These methods are unnecessary as the fact that this trip belongs to a driver and belongs to a passenger ensure the methods exist already (provided by rails). 

```suggestion
```